tests/controls/security_test.rb

```diff
@@ -0,0 +1,52 @@
+control "internal-security-1.0" do
+  impact 1.0
+  title "Tester for Openstack security rules."
+  desc "put description here"
+
+  username = user.username
+
+  describe sshd_config do
+    its('PermitRootLogin') { should eq 'no' }
+    its('PasswordAuthentication') { should eq 'no' }
+    its('ChallengeResponseAuthentication') { should eq 'no' }
+    its('KbdInteractiveAuthentication') { should eq 'no' }
+  end
+
+  describe 'running sshd config' do
+    let(:resource) { command('sudo sshd -T') }
+
+    it 'should not permit root login' do
+      expect(resource.stdout).to cmp /^PermitRootLogin no/i
+    end
+
+    it 'should not permit password authentication' do
+      expect(resource.stdout).to cmp /^PasswordAuthentication no/i
+    end
+
+    it 'should not permit challenge response authentication' do
+      expect(resource.stdout).to cmp /^ChallengeResponseAuthentication no/i
+    end
+    it 'should not permit keyboard interactive authentication' do
+      expect(resource.stdout).to cmp /^KbdInteractiveAuthentication no/i
+    end
+  end
+
+=begin
+  Our version of inspec does not give us a warning about the list matcher,
+  but in future versions of inspec this will be removed.
```

Comment:
I tried using entries, but it kicked back an error. It seems that our version is from before they decided to move away from `list`.